src/components/Range.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  label {
+    margin-right: 10px;
+  }
+  output {
+    background: white;
+    margin: 10px;
+    padding: 6px 10px;
+    border-radius: 6px;
+    border: 1px solid ${({ theme }) => theme.border};
+  }
+`;
+const Input = styled.input`
+  align-items: center;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background: none;
+  border: none;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
```

Comment:
You've already got a `align-items` above.

Reply:
Done